actors/common/src/test/java/org/sunbird/feed/impl/FeedServiceImplTest.java

```diff
@@ -0,0 +1,140 @@
+import org.sunbird.cassandra.CassandraOperation;
+import org.sunbird.cassandraimpl.CassandraOperationImpl;
+import org.sunbird.common.Constants;
+import org.sunbird.common.ElasticSearchHelper;
+import org.sunbird.common.ElasticSearchRestHighImpl;
+import org.sunbird.common.factory.EsClientFactory;
+import org.sunbird.common.inf.ElasticSearchService;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.feed.IFeedService;
+import org.sunbird.helper.ServiceFactory;
+import org.sunbird.models.user.Feed;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({
+  ServiceFactory.class,
+  ElasticSearchRestHighImpl.class,
+  ElasticSearchHelper.class,
+  EsClientFactory.class,
+  CassandraOperationImpl.class,
+  ElasticSearchService.class,
+  FeedServiceImpl.class,
+})
+@SuppressStaticInitializationFor("org.sunbird.common.ElasticSearchUtil")
+@PowerMockIgnore({"javax.management.*"})
+public class FeedServiceImplTest {
+  private ElasticSearchService esUtil;
+  private static CassandraOperation cassandraOperation = null;
+  private static IFeedService feedService = FeedFactory.getInstance();
+
+  @Before
+  public void setUp() throws Exception {
+    PowerMockito.mockStatic(ServiceFactory.class);
+    PowerMockito.mockStatic(EsClientFactory.class);
+    PowerMockito.mockStatic(ElasticSearchHelper.class);
+    PowerMockito.mockStatic(FeedServiceImpl.class);
+    esUtil = mock(ElasticSearchService.class);
+    esUtil = mock(ElasticSearchRestHighImpl.class);
+    when(EsClientFactory.getInstance(Mockito.anyString())).thenReturn(esUtil);
+    cassandraOperation = mock(CassandraOperationImpl.class);
+    PowerMockito.when(ServiceFactory.getInstance()).thenReturn(cassandraOperation);
+    when(FeedServiceImpl.getCassandraInstance()).thenReturn(cassandraOperation);
+    when(FeedServiceImpl.getESInstance()).thenReturn(esUtil);
+  }
+
+  private static void initCassandraForSuccess() {
+    Response response = new Response();
+    Map<String, Object> responseMap = new HashMap<>();
+    responseMap.put(Constants.RESPONSE, Arrays.asList(getFeedMap()));
+    response.getResult().putAll(responseMap);
+    PowerMockito.when(
+            cassandraOperation.getRecordsByProperties(Mockito.any(), Mockito.any(), Mockito.any()))
+        .thenReturn(response);
+
+    Response upsertResponse = new Response();
+    Map<String, Object> responseMap2 = new HashMap<>();
+    responseMap2.put(Constants.RESPONSE, Constants.SUCCESS);
+    upsertResponse.getResult().putAll(responseMap2);
+    PowerMockito.when(cassandraOperation.upsertRecord(Mockito.any(), Mockito.any(), Mockito.any()))
+        .thenReturn(upsertResponse);
+    PowerMockito.when(
+            cassandraOperation.deleteRecord(
+                Mockito.anyString(), Mockito.anyString(), Mockito.anyString()))
+        .thenReturn(upsertResponse);
+  }
+
+  @Test
+  public void testInsert() {
+    initCassandraForSuccess();
```

Comment:
initCassandraForSuccess() method can be called from setup() last line. in that case we don't need to call from each method.